### ISSUE TITLE
docker-quickstart-terminal or Kitematic fail to create a functional default VM

### ISSUE THREAD

**Issue by glw** [docker/toolbox#229]
Im on Windows 7 64 bit, with VT-X on
Docker Toolbox install version is 1.8.2c

When starting Kitematic without first starting Docker QuickStart Terminal I get error:
"Windows has detected and IP address conflict"
Then it hangs at 99% complete for a while, then i get an error:

```
C:\Program Files\Docker Toolbox\kitematic\resources\resources\docker-machine.exe -D create -d virtualbox --virtualbox-memory 2048 default returned non zero exit code. Stderr: Converting from raw image file="stdin" to file="C:\Users\<redacted>\disk.vmdk"... Creating dynamic image with size 20971520000 bytes (20000MB)... Error creating machine: Maximum number of retries (60) exceeded You will want to check the provider to make sure the machine and associated resources were properly removed.
```
When Starting with Docker QuickStart Terminal first, It gets through:
```
    Creating Machine default...
    Creating VirtualBox VM...
    Creating SSH key...
    Starting VirtualBox VM...
    Starting VM...
```
Then after a while:
```    
Error creating machine: Maximum number of retries (5) exceeded
    You will want to check the provider to make sure the machine and associated resources were properly     removed.
Looks like something went wrong... Press any key to continue...
```
Either way I attempt I do end up with a VM but it is broken.

I have tried VirtualBox versions 4.13, 5.0, and testing none have worked so far.

Using Docker-machine 

```
restart
``` 

outputs: 

```
Restarting machines may have new IP addresses. You may need to re-run the 'dokcer-machine env' command.
```

running env 

gives me

```
open c:\Users\Username\.docker\machine\machines\default\ca.pem: The system cannot find the file specified.
```

When looking into my VBlog for the default VM I see this at the end:
(sorry github could not attach a txt file for some reason)

```
00:00:01.070849 PIT: mode=2 count=0x48d3 (18643) - 64.00 Hz (ch=0)
00:00:01.093774 PIT: mode=2 count=0x10000 (65536) - 18.20 Hz (ch=0)
00:00:01.094602 Guest Log: BIOS: Boot : bseqnr=1, bootseq=0233
00:00:01.096268 Guest Log: BIOS: Booting from CD-ROM...
00:00:01.433078 Guest Log: BIOS: KBD: unsupported int 16h function 03
00:00:01.433872 Guest Log: BIOS: AX=0305 BX=0000 CX=0000 DX=0000 
00:00:07.881609 AHCI#0: Reset the HBA
00:00:07.888003 AHCI#0: Port 0 reset
00:00:08.236775 AHCI#0: Port 1 reset
00:00:20.573427 Guest Additions information report: Version 5.0.4 r102546 '5.0.4'
00:00:20.573482 Guest Additions information report: Interface = 0x00010004 osType = 0x00053100
00:00:20.573576 Guest Additions capability report: (0x0 -> 0x0) seamless: no, hostWindowMapping: no, graphics: no
00:00:20.573678 Guest reported fixed hypervisor window at 00000c00000 (size = 0xc00000, rc = VINF_SUCCESS)
00:00:20.573888 Guest Log: vboxguest: misc device minor 56, IRQ 20, I/O port d020, MMIO at 00000000f0400000 (size 0x400000)
00:00:25.565334 NAT: link up
00:00:28.541370 Guest Log: VBoxService 5.0.4 r102546 (verbosity: 0) linux.amd64 (Sep  8 2015 09:05:09) release log
00:00:28.541416 Guest Log: 00:00:00.000229 main     Log opened 2015-10-08T21:36:52.150834000Z
00:00:28.547426 Guest Log: 00:00:00.006494 main     OS Product: Linux
00:00:28.547536 Guest Log: 00:00:00.006614 main     OS Release: 4.0.9-boot2docker
00:00:28.547646 Guest Log: 00:00:00.006724 main     OS Version: #1 SMP Thu Sep 10 20:39:20 UTC 2015
00:00:28.547727 Guest Log: 00:00:00.006799 main     OS Service Pack: #1 SMP Thu Sep 10 20:39:20 UTC 2015
00:00:28.547809 Guest Log: 00:00:00.006880 main     Executable: /sbin/VBoxService
00:00:28.547826 Guest Log: 00:00:00.006890 main     Process ID: 854
00:00:28.547842 Guest Log: 00:00:00.006893 main     Package type: LINUX_64BITS_GENERIC
00:00:28.555368 Guest Log: 00:00:00.014076 main     5.0.4 r102546 started. Verbose level = 0
00:00:28.584826 Guest Log: 00:00:00.043457 vminfo   rtldrNativeLoad: dlopen('libdbus-1.so.3', RTLD_NOW | RTLD_LOCAL) failed: libdbus-1.so.3: cannot open shared object file: No such file or directory
00:00:28.584966 Guest Log: 00:00:00.043635 vminfo   Error: Unable to connect to system D-Bus (1/3): D-Bus not installed
00:00:28.693485 NAT: IPv6 not supported
00:00:29.381399 NAT: DNS#0: 10.128.50.187
00:00:29.381441 NAT: DNS#1: 10.128.50.186
00:00:29.381451 NAT: DNS#2: 10.128.51.100
00:00:29.381458 NAT: DNS#3: 10.128.51.106
00:00:29.381471 NAT: DHCP offered IP address 10.0.2.15
00:00:29.382153 NAT: DHCP offered IP address 10.0.2.15
00:00:33.589032 Guest Log: 00:00:05.049854 vminfo   Error: Unable to connect to system D-Bus (2/3): D-Bus not installed
00:00:38.593880 Guest Log: 00:00:10.057548 vminfo   Error: Unable to connect to system D-Bus (3/3): D-Bus not installed
```
Could the missing D-Bus be he reason for all all the errors?

Sorry if this is a duplicate, its a little hard to follow working solutions and current status through the issues page.

Thanks for the help.

**Comment by ObjectIsAdvantag** [comment]
+1

**Comment by ObjectIsAdvantag** [comment]
Seems that is has been fixed, waiting for the next build...

see 
- https://github.com/docker/machine/pull/1674 for the fix
- and https://github.com/docker/machine/issues/1573 for detailled info about the issue

**Comment by DimitrK** [comment]
Same behavior here. Awaiting next build.

**Comment by jippi** [comment]
when is next build?

**Comment by glw** [comment]
according to the link for issue 1573 above, it'll be in 2 weeks. and that was posted 11 days ago.

**Comment by mchiang0610** [comment]
@jippi Next build should come out with the rest of Docker 1.9 

This issue is related to Docker machine, and using Virtualbox.

**Comment by mchiang0610** [comment]
Merging this with #153

**Comment by ObjectIsAdvantag** [comment]
Fix confirmed, no more IP address conflict
- Windows 7 64bits laptop, 
- DockerToolbox 1.9 RC2: https://github.com/docker/toolbox/releases/tag/v1.9.0-rc2
- VirtualBox 5.0.8 
- docker-machine create from the CMD (windows shell)

Side note : boot2docker is still issuing the "Unable to connect to system D-Bus" message

**Comment by chriscalip** [comment]
Bug Appearance after alleged fix.

Creating Machine default...
Creating VirtualBox VM...
Creating SSH key...
Error creating machine: exit status 1
You will want to check the provider to make sure the machine and associated reso                                                                                                                urces were properly removed.
Looks like something went wrong... Press any key to continue...

Then installed 
https://github.com/docker/toolbox/releases/tag/v1.9.0-rc2

Got this error message.
Error creating machine: Error in driver during machine creation: exit status 1
Looks like something went wrong... Press any key to continue...

**Comment by chriscalip** [comment]
Tried to run kitematic:

C:\Program Files\Docker Toolbox\docker-machine.exe -D create -d virtualbox --virtualbox-memory 2048 default returned non zero exit code. Stderr: Docker Machine Version: 0.5.0-rc3 (a1e610b) Launching plugin server for driver virtualbox Found binary path at .\docker-machine-driver-virtualbox.exe Plugin server listening at address 127.0.0.1:49764 () Got msg {ServiceMethod:RpcServerDriver.GetVersion Args:{} Reply:0xc0820f4100} Using API Version 1 () Got msg {ServiceMethod:RpcServerDriver.SetConfigRaw Args:[123 34 73 80 65 100 100 114 101 115 115 34 58 34 34 44 34 83 83 72 85 115 101 114 34 58 34 34 44 34 83 83 72 80 111 114 116 34 58 48 44 34 77 97 99 104 105 110 101 78 97 109 101 34 58 34 50 48 52 56 34 44 34 83 119 97 114 109 77 97 115 116 101 114 34 58 102 97 108 115 101 44 34 83 119 97 114 109 72 111 115 116 34 58 34 34 44 34 83 119 97 114 109 68 105 115 99 111 118 101 114 121 34 58 34 34 44 34 83 116 111 114 101 80 97 116 104 34 58 34 34 125] Reply:<nil>} () Got msg {ServiceMethod:RpcServerDriver.GetMachineName Args:{} Reply:0xc0820f51b0} (2048) Got msg {ServiceMethod:RpcServerDriver.GetCreateFlags Args:{} Reply:0xc082104a80} Launching plugin server for driver virtualbox Found binary path at .\docker-machine-driver-virtualbox.exe Plugin server listening at address 127.0.0.1:49768 () Got msg {ServiceMethod:RpcServerDriver.GetVersion Args:{} Reply:0xc08218e0b0} Using API Version 1 () Got msg {ServiceMethod:RpcServerDriver.SetConfigRaw Args:[123 34 73 80 65 100 100 114 101 115 115 34 58 34 34 44 34 83 83 72 85 115 101 114 34 58 34 34 44 34 83 83 72 80 111 114 116 34 58 48 44 34 77 97 99 104 105 110 101 78 97 109 101 34 58 34 100 101 102 97 117 108 116 34 44 34 83 119 97 114 109 77 97 115 116 101 114 34 58 102 97 108 115 101 44 34 83 119 97 114 109 72 111 115 116 34 58 34 34 44 34 83 119 97 114 109 68 105 115 99 111 118 101 114 121 34 58 34 34 44 34 83 116 111 114 101 80 97 116 104 34 58 34 67 58 92 92 85 115 101 114 115 92 92 99 104 114 105 115 99 97 108 105 112 92 92 46 100 111 99 107 101 114 92 92 109 97 99 104 105 110 101 34 125] Reply:<nil>} () Got msg {ServiceMethod:RpcServerDriver.GetMachineName Args:{} Reply:0xc08218f160} (default) Got msg {ServiceMethod:RpcServerDriver.GetMachineName Args:{} Reply:0xc0820f5310} (default) Got msg {ServiceMethod:RpcServerDriver.DriverName Args:{} Reply:0xc0820f53c0} (default) Got msg {ServiceMethod:RpcServerDriver.GetCreateFlags Args:{} Reply:0xc082104be0} (default) Got msg {ServiceMethod:RpcServerDriver.SetConfigFromFlags Args:0xc0820f5d90 Reply:<nil>} (default) Got msg {ServiceMethod:RpcServerDriver.PreCreateCheck Args:{} Reply:<nil>} (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe (default) Got msg {ServiceMethod:RpcServerDriver.GetConfigRaw Args:{} Reply:0xc08215ce40} (default) Got msg {ServiceMethod:RpcServerDriver.GetConfigRaw Args:{} Reply:0xc08215d240} (default) Got msg {ServiceMethod:RpcServerDriver.Create Args:{} Reply:<nil>} (default) DBG | COMMAND: wmic cpu get VirtualizationFirmwareEnabled (default) OUT | Creating VirtualBox VM... (default) DBG | Couldn't check that VT-X/AMD-v is enabled. Will check that the vm is properly created: exit status 2147749911 (default) OUT | Creating SSH key... (default) DBG | Creating disk image... (default) DBG | Creating 20000 MB hard disk image... (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe createvm --basefolder C:\Users\chriscalip\.docker\machine\machines\default --name default --register (default) DBG | STDOUT: (default) DBG | { (default) DBG | Virtual machine 'default' is created and registered. (default) DBG | UUID: 2684f9ea-ca98-44a3-9e23-f72ef88a7ea5 (default) DBG | Settings file: 'C:\Users\chriscalip\.docker\machine\machines\default\default\default.vbox' (default) DBG | } (default) DBG | STDERR: (default) DBG | { (default) DBG | } (default) DBG | VM CPUS: 1 (default) DBG | VM Memory: 2048 (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --firmware bios --bioslogofadein off --bioslogofadeout off --bioslogodisplaytime 0 --biosbootmenu disabled --ostype Linux26_64 --cpus 1 --memory 2048 --acpi on --ioapic on --rtcuseutc on --natdnshostresolver1 off --natdnsproxy1 off --cpuhotplug off --pae on --hpet on --hwvirtex on --nestedpaging on --largepages on --vtxvpid on --accelerate3d off --boot1 dvd (default) DBG | STDOUT: (default) DBG | { (default) DBG | } (default) DBG | STDERR: (default) DBG | { (default) DBG | } (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic1 nat --nictype1 82540EM --cableconnected1 on (default) DBG | STDOUT: (default) DBG | { (default) DBG | } (default) DBG | STDERR: (default) DBG | { (default) DBG | } (default) DBG | using 192.168.99.6 for dhcp address (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs (default) DBG | STDOUT: (default) DBG | { (default) DBG | Name: VirtualBox Host-Only Ethernet Adapter (default) DBG | GUID: a05a5783-11f9-4213-994b-39b7c5e558e3 (default) DBG | DHCP: Enabled (default) DBG | IPAddress: 169.254.137.228 (default) DBG | NetworkMask: 255.255.0.0 (default) DBG | IPV6Address: fe80:0000:0000:0000:8c44:d577:5775:89e4 (default) DBG | IPV6NetworkMaskPrefixLength: 64 (default) DBG | HardwareAddress: 08:00:27:00:dc:93 (default) DBG | MediumType: Ethernet (default) DBG | Status: Up (default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter (default) DBG | (default) DBG | } (default) DBG | STDERR: (default) DBG | { (default) DBG | } (default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe hostonlyif create (default) DBG | STDOUT: (default) DBG | { (default) DBG | } (default) DBG | STDERR: (default) DBG | { (default) DBG | 0%... (default) DBG | Progress state: E_FAIL (default) DBG | VBoxManage.exe: error: Failed to create the host-only adapter (default) DBG | VBoxManage.exe: error: SetupDiCallClassInstaller (DIF_INSTALLDEVICE) failed (0x0000000D) (default) DBG | VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component HostNetworkInterfaceWrap, interface IHostNetworkInterface (default) DBG | VBoxManage.exe: error: Context: "enum RTEXITCODE __cdecl handleCreate(struct HandlerArg *)" at line 71 of file VBoxManageHostonly.cpp (default) DBG | } Making call to close driver server (default) Got msg {ServiceMethod:RpcServerDriver.Close Args:{} Reply:<nil>} Making call to close driver server (2048) Got msg {ServiceMethod:RpcServerDriver.Close Args:{} Reply:<nil>} Successfully made call to close driver server Error closing connection to plugin server: read tcp 127.0.0.1:49769->127.0.0.1:49768: wsarecv: An existing connection was forcibly closed by the remote host. Error creating machine: Error in driver during machine creation: exit status 1

**Comment by yoshz** [comment]
The problem still exists in 1.9.1a!

**Comment by nodeit** [comment]
@yoshz still getting the same error as well.

**Comment by dgangwar** [comment]
I am also getting the below error when creating a new machine in the end of VBox.log.

Docker]# docker-machine -v
docker-machine version 0.5.0 (04cfa58)
Docker]# docker -v
Docker version 1.9.1, build a34a1d5

ERROR --------------

 00:00:42.132750 VMMDev: Guest Log: VBoxService 5.0.10 r104061 (verbosity: 0) linux.amd64 (Nov 10 2015 17:56:08) release log
00:00:42.132770 VMMDev: Guest Log: 00:00:00.000123 main     Log opened 2015-12-12T21:33:32.572454000Z
00:00:42.139354 VMMDev: Guest Log: 00:00:00.006744 main     OS Product: Linux
00:00:42.139448 VMMDev: Guest Log: 00:00:00.006845 main     OS Release: 4.1.13-boot2docker
00:00:42.139533 VMMDev: Guest Log: 00:00:00.006926 main     OS Version: #1 SMP Fri Nov 20 19:05:50 UTC 2015
00:00:42.139619 VMMDev: Guest Log: 00:00:00.007015 main     Executable: /sbin/VBoxService
00:00:42.139623 VMMDev: Guest Log: 00:00:00.007016 main     Process ID: 2044
00:00:42.139624 VMMDev: Guest Log: 00:00:00.007017 main     Package type: LINUX_64BITS_GENERIC
00:00:42.182537 VMMDev: Guest Log: 00:00:00.049931 main     5.0.10 r104061 started. Verbose level = 0
00:00:42.218227 VMMDev: Guest Log: 00:00:00.085616 vminfo   rtldrNativeLoad: dlopen('libdbus-1.so.3', RTLD_NOW | RTLD_LOCAL) failed: libdbus-1.so.3: cannot open shared object file: No such file or directory
00:00:42.218328 VMMDev: Guest Log: 00:00:00.085728 vminfo   Error: Unable to connect to system D-Bus (1/3): D-Bus not installed
00:00:42.537879 NAT: IPv6 not supported
00:00:47.296524 VMMDev: Guest Log: 00:00:05.166594 vminfo   Error: Unable to connect to system D-Bus (2/3): D-Bus not installed
00:00:52.305453 VMMDev: Guest Log: 00:00:10.178318 vminfo   Error: Unable to connect to system D-Bus (3/3): D-Bus not installed

**Comment by jopfre** [comment]
also getting this error on windows 7
Docker version 1.9.1, build a34a1d5
Error: Unable to connect to system D-Bus (1/3): D-Bus not installed

**Comment by Yi-G** [comment]
Get the same error "D-Bus not installed" from Docker version 1.9.1, build a34a1d5

**Comment by dimroc** [comment]
Same issue:

Docker version 1.8.2, build 0a8c2e3
docker-machine version 0.5.5, build
VirtualBox Version 5.0.12 r104815

**Comment by msangel** [comment]
DockerToolbox-1.9.1g
VirtualBox Version 5.0.12

same here

**Comment by jycr** [comment]
Like @msangel, I've the same issue.
I run DockerToolbox-1.9.1g (with VirtualBox Version 5.0.12)

I have in VBox.log :
```
00:00:36.838771 NAT: Link up
00:00:51.785365 NAT: DHCP offered IP address 10.0.2.15
00:00:51.785790 NAT: DHCP offered IP address 10.0.2.15
00:00:54.017111 VMMDev: Guest Log: VBoxService 5.0.10 r104061 (verbosity: 0) linux.amd64 (Nov 10 2015 17:56:08) release log
00:00:54.017229 VMMDev: Guest Log: 00:00:00.000275 main     Log opened 2016-01-07T12:59:42.456461000Z
00:00:54.025455 VMMDev: Guest Log: 00:00:00.008973 main     OS Product: Linux
00:00:54.025642 VMMDev: Guest Log: 00:00:00.009178 main     OS Release: 4.1.13-boot2docker
00:00:54.025788 VMMDev: Guest Log: 00:00:00.009327 main     OS Version: #1 SMP Fri Nov 20 19:05:50 UTC 2015
00:00:54.025932 VMMDev: Guest Log: 00:00:00.009466 main     Executable: /sbin/VBoxService
00:00:54.025985 VMMDev: Guest Log: 00:00:00.009471 main     Process ID: 1990
00:00:54.026018 VMMDev: Guest Log: 00:00:00.009473 main     Package type: LINUX_64BITS_GENERIC
00:00:54.038575 VMMDev: Guest Log: 00:00:00.021759 main     5.0.10 r104061 started. Verbose level = 0
00:00:54.060248 VMMDev: Guest Log: 00:00:00.043416 vminfo   rtldrNativeLoad: dlopen('libdbus-1.so.3', RTLD_NOW | RTLD_LOCAL) failed: libdbus-1.so.3: cannot open shared object file: No such file or directory
00:00:54.060487 VMMDev: Guest Log: 00:00:00.043685 vminfo   Error: Unable to connect to system D-Bus (1/3): D-Bus not installed
00:00:54.854400 NAT: IPv6 not supported
00:00:59.063835 VMMDev: Guest Log: 00:00:05.049275 vminfo   Error: Unable to connect to system D-Bus (2/3): D-Bus not installed
00:01:04.088808 VMMDev: Guest Log: 00:00:10.076681 vminfo   Error: Unable to connect to system D-Bus (3/3): D-Bus not installed
```

What does this line means??? :
```
00:00:54.060248 VMMDev: Guest Log: 00:00:00.043416 vminfo   rtldrNativeLoad: dlopen('libdbus-1.so.3', RTLD_NOW | RTLD_LOCAL) failed: libdbus-1.so.3: cannot open shared object file: No such file or directory
```

**Comment by msangel** [comment]
I have removed my comment just few seconds ago because this bug is already "closed", but problem is present. I planing to create new one

**Comment by ArchiMageAlex** [comment]
Please reopen this issue, behaviour of last version same as at begining.

**Comment by raagav-iyer** [comment]
I faced the same issue.
It was working like a clock and all of a sudden it stopped.
I back tracked what I did. I figured out the I had installed OpenSSH for windows, since when it stopped.
I uninstalled OpenSSH and it started working like a clock again.

**Comment by schoolAct1** [comment]
I faced same issue today (Win7 Docker 1.9.1).  I resolved issue by creating new default machine `docker-machine create defaultnew` then copying the certs from the new machine `/var/lib/boot2docker/*pem` to same path on broken machine.  I also copied the certs to local Windows path C:\Users\<user>\.docker\machine\machines\default\.  You can use docker-machine scp command to accomplish this

I then ran `docker-machine restart default` and received error:

`Error checking TLS connection: Error checking and/or regenerating the certs: There was an error validating certificates for host "192.1.1.100:2376": x509: certificate is valid for 192.1.1.101, n
ot 192.168.99.100
You can attempt to regenerate them using 'docker-machine regenerate-certs [name]'.
Be advised that this will trigger a Docker daemon restart which will stop running containers.`


This is because I copied the certs.  However, the docker daemon is now running on broken machine so this time after running `docker-machine regenerate-certs default` it was successful and regenerated the certs and I now can connect back to my old machine after running:

`eval "$(docker-machine env default)"`

**Comment by eniniz** [comment]
+1
